### ISSUE TITLE
Problem remote server

### ISSUE THREAD

**Issue by amsicora68** [kr15h/ofxPiMapper#136]
Hello
I tried but it gives me this error:
/home/pi/openFrameworks/addons/ofxPiMapper/src/Sources/VideoSource.h:7:27: fatal error: ofxOMXPlayer.h: File o directory non esistente
  #include "ofxOMXPlayer.h"
the example of ofxPiMapper works fine instead.
Thanks
Marco

**Comment by kr15h** [comment]
What system are you compiling on?

**Comment by amsicora68** [comment]
raspberry 2 and 3 this error is on 2 with raspbian jessie

**Comment by kr15h** [comment]
Did you clone ofxOMXPlayer into the openFrameworks/addons folder?

**Comment by amsicora68** [comment]
Of course.The example of ofxPiMapper works fine.

**Comment by kr15h** [comment]
Weird. Will have to check this on a Pi later.

**Comment by magdesign** [comment]
Yup, can confirm on Raspian Jessie:
When trying to compile example_remote-server, following error terminates compiling:

```
/home/pi/openFrameworks/addons/ofxPiMapper/src/Sources/VideoSource.h:7:27: fatal error: ofxOMXPlayer.h: No such file or directory
  #include "ofxOMXPlayer.h"
```

**Comment by pietrondo** [comment]
you need to add in addons.config
ofxOMXPlayer

**Comment by pietrondo** [comment]
no sorry
in addons.make

**Comment by kr15h** [comment]
Ok, fixed it 2c705c0a653a283b0a9ef0290fdb06484eaa65bf

**Comment by amsicora68** [comment]
many thanks works well now!
